### ISSUE TITLE
Response as an array

### ISSUE THREAD

**Issue by qwertz1958** [mevdschee/php-crud-api#847]
Hi there,

Is there a way to get the response as an array?

I want to call a URL:
http://myServer/array/myTable?id,lt,20

As a result I want to get an array in php. Is that possible?

Kind regards

Stephen

**Comment by mevdschee** [comment]
>As a result I want to get an array in php. Is that possible?

No, not directly. The result will be a JSON string and that needs to be decoded and transformed into a PHP array.

    $json = ... // call the api using curl?
    $result = json_decode($json);
    $array = array_column($result['records'],'id');

This will retrieve an array with all the id column values.

**Comment by qwertz1958** [comment]
At the moment I'm working with the following call.

$api = new Api($config);
$response = $api->handle($request);

Which call do I have to choose to be able to use the JSON file?

Kind regards

Stephan

**Comment by mevdschee** [comment]
>Which call do I have to choose to be able to use the JSON file?

    $json = $response->getBody()->getContents();

This is PSR7, see: https://www.php-fig.org/psr/psr-7/

**Comment by mevdschee** [comment]
any update?

**Comment by mevdschee** [comment]
closing due to inactivity.
